cpp/src/Slice/Parser.cpp

```diff
@@ -1238,7 +1227,6 @@ Slice::Container::createClassDecl(const string& name)
         }
     }
 
-    _unit->currentContainer();
```

Comment:
`currentContainer` is `const`. All it does is return a reference to the current container.
So, calling this function and ignoring the return value is pointless.